book/website/netlify/_redirects

```diff
@@ -0,0 +1,3 @@
+# Handle HTTP and HTTPS redirects to the start page
+http://the-turing-way.org/*  https://the-turing-way.start.page/ 301!
+https://the-turing-way.org/* https://the-turing-way.start.page/ 301!
```

Comment:
```suggestion
http://the-turing-way.org/*  https://the-turing-way.start.page/ 302!
https://the-turing-way.org/* https://the-turing-way.start.page/ 302!
```

I think we want these to be 302 as we are not replacing `the-turing-way.org` with `the-turing-way.start.page`.

Reply:
Makes sense to me 👍🏻